ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -11,14 +11,15 @@ build_root:
 images:
 - dockerfile_literal: |-
     FROM src
+    RUN dnf -y install python3-openstackclient jq && dnf clean all
```

Comment:
it makes no difference, but is this not already installed here: 
https://github.com/openshift/installer/blob/master/images/openstack/Dockerfile.ci#L29